<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>366</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>366</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FreeFS Formatting Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="banner">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>341</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>341</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>FreeFS Formatting Tool</string>
    </property>
   </widget>
   <widget class="QComboBox" name="device">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No device found.</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="banner1">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>50</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Device</string>
    </property>
   </widget>
   <widget class="QLabel" name="banner2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>File System</string>
    </property>
   </widget>
   <widget class="QComboBox" name="fs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>351</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>FreeFS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FreeFS (Encrypted)</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="banner3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="selImage">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select image</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="banner4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Progress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="format">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>490</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #d00;</string>
    </property>
    <property name="text">
     <string>Format</string>
    </property>
   </widget>
   <widget class="QLabel" name="banner5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Label</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="diskLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="fs_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>351</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>512</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1024</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2048</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4096</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8192</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="banner7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sector size</string>
    </property>
   </widget>
   <widget class="QLabel" name="banner6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Disk size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="size">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="SizeType">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Kb</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mb</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gb</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tb</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
